trino-aws-proxy/src/main/java/io/trino/aws/proxy/server/credentials/file/FileBasedCredentialsProviderModule.java

```diff
@@ -28,5 +31,9 @@ protected void setup(Binder binder)
     {
         configBinder(binder).bindConfig(FileBasedCredentialsProviderConfig.class);
         binder.bind(CredentialsProvider.class).to(FileBasedCredentialsProvider.class).in(Scopes.SINGLETON);
+
+        // Provide addAbstractTypeMapping for Identity class
+        ObjectMapper objectMapper = new ObjectMapperProvider().get();
```

Comment:
You shouldn't bind `ObjectMapper`. Airlift already does this.

Reply:
Since, we provide explicit abstracttype mapping for the Identity class, Shouldn't we do this - https://github.com/trinodb/aws-proxy/pull/123/files#diff-a1075cd41e6683aaecf551ceeac9a8a4512680a05335e3d558066381bc335fdeR42

Reply:
I added a comment below. That shouldn't be needed.

Reply:
If we add a default method to `Identity`, though, we don't need anything special.

Reply:
We don't want annotated ObjectMappers. This isn't supportable. You should put your module in the global ObjectMapperProvider